templates/python/custom-copilot-rag-customize/teamsapp.local.yml.tpl

```diff
@@ -0,0 +1,80 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.3/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.3
+
+provision:
+  # Creates a Teams app
+  - uses: teamsApp/create
+    with:
+      # Teams app name
+      name: {{appName}}${{TEAMSFX_ENV}}
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile: 
+      teamsAppId: TEAMS_APP_ID
+
+  # Create or reuse an existing Azure Active Directory application for bot.
+  - uses: botAadApp/create
```

Comment:
Please use `aadApp/create` to create an AAD app for bot. You can refer other templates about how to do it.

Reply:
Please also update the yaml file for the azure ai search template.